trtools/mergeSTR/mergeSTR.py

```diff
@@ -39,6 +39,7 @@
     trh.VcfTypes.advntr: [("END", True), ("VID", True), ("RU", True), ("RC", True)]
 }
 
+#BEAGLEINFOFIELDS = ["IMP"]
```

Comment:
if we are not using this we can just remove all the commented lines about the IMP field